src/main/java/hudson/plugins/parameterizedtrigger/BuildTriggerConfig.java

```diff
@@ -412,9 +412,20 @@ private static void resolveProject(AbstractBuild build, SubProjectData subProjec
 			// don't trigger on this configuration
 		}
         return Collections.emptyList();
-	}
-
+    }
 
+    private void reportSchedulingError(@Nonnull Run<?, ?> run, @Nonnull Job<?, ?> jobToTrigger, @Nonnull BuildListener listener) {
+        // Do not print details to Build Listener, they have been reported previously in #canTriggerProject()
+        listener.error("Skipping " + jobToTrigger.getFullName() + "...");
+        if (LOGGER.isLoggable(Level.CONFIG)) {
```

Comment:
`BuildTrigger` and `ReverseBuildTrigger` print similar messages at `WARNING`. Is there a reason you are suppressing this message from logs by default? It is unlikely an admin will ever see it—by the time they figure out they need to be turning on this logger, they will already know about the problem and the fix. The point is to guide admins to help their users address configuration problems.